app/views/work_packages/show.html.erb

```diff
@@ -52,7 +52,7 @@ See doc/COPYRIGHT.rdoc for more details.
   <%= render partial: 'show_attributes', locals: { work_package: work_package } %>
 
   <% if work_package.description? %>
-    <hr />
+    <hr>
```

Comment:
? This would belong to typography and i removed it from the condensed list of quick wins (see commentary in the ticket)